pom.xml

```diff
@@ -543,6 +543,11 @@
       <artifactId>junit-vintage-engine</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit-pioneer</groupId>
+      <artifactId>junit-pioneer</artifactId>
+      <version>0.2.0</version>
```

Comment:
This dependency should have `test` scope.

Reply:
Good catch, thanks.